cloudtrail-lambda-dynamo-cdk/README.md

```diff
@@ -29,6 +28,10 @@ s.
     ```
     npm install
     ```
+1. Open the lambda function object_tag_checker.py, and on line 40 replace the example keys with your required keys:
+    ```
+    required_keys = {"Key1", "Key2", "Key3", "Key4"}
```

Comment:
what are these for? can you elaborate more or give some examples or links to docs of what you are doing